### PR TITLE
 remove readonly prop from select's input

### PR DESCRIPTION
**Pull request by l3satwik**

### CODE REVIEW

File: src/components/Select.tsx [hunk 1]
```diff
@@ -53,7 +53,6 @@ const Select: React.SFC<SelectProps> = props => {
                 required={required}
                 message={isOpen ? " " : ""}
                 errorMessage={errorMessage}
-                readOnly
```
Comment (l3satwik) [inline src/components/Select.tsx]: @ritz078 It made the font color gray, which was not according to the design system.

**Comment by ritz078** [comment]
tests are breaking.

**Comment by azizhk** [comment]
Sorry did not see this.
Also did same in https://github.com/ritz078/pebble/pull/103.
Did not want to remove readonly as might be focusable using tab.

**Comment by azizhk** [comment]
Fixed in #103

**Pull request closed without merging**